### PR TITLE
141617753 total calories meal aw

### PR DESCRIPTION
**Pull request by wlffann**
This PR adds each table's total calories and remaining calories based on the foods in the meal object. It is linked to the date function so that it displays only the calories for the day's meals' foods. 

@case-eee @icorson3 @DavidKnott @jdconrad89

### CODE REVIEW

File: lib/date.js [hunk 1]
```diff
@@ -4,19 +4,26 @@ $('document').ready(function(){
   localStorage.setItem("currentDate", JSON.stringify(todayDate.format("MMMM Do, YYYY")));
   currentDate = todayDate;
   add_day();
+  addCaloriesToTable();
+  totalsTable();
```
Comment (jdconrad89) [inline lib/date.js:8]: @wlffann Thanks for adding these for me!

**Comment by jdconrad89** [comment]
@wlffann looks good to me! Love the helper in the tests!

File: lib/total-calories.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  var totalCalories = 0
+  $.grep(meal, function(foodData) {
+    totalCalories += Number(foodData.calories);
+  });
+  return totalCalories;
+};
+
+function calculateMealsRemainingCalories(totalCalories) {
+  return 200 - totalCalories;
+};
```
Comment (DavidKnott) [inline lib/total-calories.js]: Good job catching the 200!

**Comment by DavidKnott** [comment]
Looks good to merge after the 200 fix.

**Comment by DavidKnott** [comment]
Thanks for making those changes!

**Pull request merged**